### DISCUSSION TITLE
customScalingRunningJobPercentage description

### DISCUSSION THREAD

**Discussion by ra002890** [kedacore/keda#1316] [Q&A / Need Help]
Hello guys, how are you doing?

Could you explain better what customScalingRunningJobPercentage parameter do or represents?
Another question: Is it possible to use scalingStrategy on ScaledObject? If not, why? I am asking it because I would like to scale my deployment based on each 3 messages on rabbit queue. Like if I have 3 messages on queue, just one replica, if it grows between 4 and 6 messages one more replica, and if it grows between 7 and 9 I would like to have 3 replicas and so on until a maximun of 10 replicas. Is there a way to do  that now a days?

Thanks, and congratulations for the excellent work ...

**Comment by tomkerkhove** [comment]
Maybe @TsuyoshiUshio can help here and we should improve the docs if we can
